app/src/main/java/com/ivy/wallet/ui/statistic/level2/ItemStatisticScreen.kt

```diff
@@ -158,6 +164,11 @@ fun BoxWithConstraintsScope.ItemStatisticScreen(screen: ItemStatistic) {
         account = account,
         category = category,
 
+
+        accountNameConfirmation = accountNameConfirmation,
+        updateAccountNameConfirmation = { viewModel.updateAccountDeletionState(it) },
```

Comment:
nit: I guess you use:
`        updateAccountNameConfirmation = viewModel::updateAccountDeletionState,`